cukes-ldap/src/main/java/lv/ctco/cukes/ldap/facade/EntityFacade.java

```diff
@@ -80,9 +82,17 @@ private Attribute getNotNullAttribute(String attribute) {
         return attr;
     }
 
-    public void entityHasAttributeWithValue(String attribute, String value) {
-        Attribute attr = getNotNullAttribute(attribute);
-        assertThat("Should have attribute '" + attribute + "' with value '" + value + "'", attr.contains(value), is(true));
+    public void entityHasAttributeWithValue(String expectedAttr, String expectedValue) {
+        Attribute actualAttr = getNotNullAttribute(expectedAttr);
+        List<String> attributesList = new ArrayList<>();
+        for(int i = 0; i < actualAttr.size(); i++){
+            try {
+                attributesList.add(actualAttr.get(i).toString());
+            } catch (NamingException e) {
+                e.printStackTrace();
```

Comment:
Avoid using printStackTrace().
Either handle exception properly in-place or throw it further